CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [4.1.0]
```

Comment:
a breaking change in a beta feature did not seem worth a major release to me, but also too big to warrant a patch one... WTDY?

Reply:
I think this can qualify more as a fix than a new feature, so I would do 4.0.2. I agree with not releasing a new major.

BTW, remember to update the package.json :+1:

Reply:
Thanks, yes I was waiting for the version to update the `package.json` .

Happy to go with `4.0.2` 👍